vm/src/stdlib/os.rs

```diff
@@ -1228,6 +1228,24 @@ mod _os {
         Ok(())
     }
 
+    #[cfg(unix)]
+    #[pyfunction]
+    fn getloadavg(vm: &VirtualMachine) -> PyResult<PyTupleRef> {
```

Comment:
You can actually return a `(f64, f64, f64)` tuple in rust, it will get automatically converted to a tuple.

Reply:
This is so cool. It is now in the code! Thanks for reviewing!